contracts/Market.sol

```diff
@@ -136,35 +124,32 @@ contract Market is
   Certificate private _certificate;
 
   /**
-   * @notice The IERC20WithPermit token used to purchase from this market.
+   * @notice Deprecated.
    */
-  IERC20WithPermit private _purchasingToken;
+  address private _purchasingToken;
```

Comment:
I figured we might leave these storage variables in place if we were to perform these changes to protect against upgrade-related incompatibilities